.travis.yml

```diff
@@ -37,11 +37,11 @@ before_install:
 
 # Install packages with conda, then pip
 install:
-  - if [[ "$COVERAGE" == "true" ]]; then
-      export PIP_DEPS="$PIP_DEPS pytest-cov";
-    elif [[ "$STATIC" == "true" ]]; then
+  - if [[ "$STATIC" == "true" ]]; then
```

Comment:
It wasn't the else here originally; you had it first. I just changed it back. I'm fine having it be the else (i.e. test STATIC == false), I just want it to be the same in both install and script.